src/main/java/com/linkedin/kmf/apps/MirrorPipelineMonitor.java

```diff
@@ -70,6 +70,7 @@ public MirrorPipelineMonitor(Map<String, Object> props, String name) throws Exce
                                                                      String topic,
                                                                      String name) throws Exception {
     Map<String, Object> topicManagementProps = createTopicManagementServiceProps(props, topic);
+    topicManagementProps.put(CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG, "");
```

Comment:
So we prematurely create the TopicManagementServiceConfig as you requested in the previous commit so that we can retrieve the partitionsToBrokerRatio to determine the number of partitions we want to create. Because we create it prematurely, we don't have access to the zk url yet. 

I agree that broker count needs to be managed dynamically and there is a TODO to do this. That is not what this commit is regarding though.